src/main/java/pl/winiarczyk/lesson4/homework/Numbers/EvenAndOddNumbers.java

```diff
@@ -0,0 +1,26 @@
+package pl.winiarczyk.lesson4.homework.Numbers;
+
+public class EvenAndOddNumbers {
+    public static int evenNumbers(int checkNumber) {
```

Comment:
spodziewałbym się bardziej, że metoda będzie zwracać true/false niż int. Bo co mi powie to, że jak przekażę do metody 2 to dostanę 2?